Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/EditorMenu.cpp

```diff
@@ -47,6 +49,38 @@ namespace AzToolsFramework
         m_menuItems.insert({ sortKey, MenuItem(widget) });
         RefreshMenu();
     }
+    
+    bool EditorMenu::ContainsAction(const AZStd::string& actionIdentifier) const
+    {
+        return m_actionToSortKeyMap.contains(actionIdentifier);
+    }
+
+    bool EditorMenu::ContainsSubMenu(const AZStd::string& menuIdentifier) const
+    {
+        return m_subMenuToSortKeyMap.contains(menuIdentifier);
+    }
+
+    int EditorMenu::GetActionSortKey(const AZStd::string& actionIdentifier) const
+    {
+        auto actionIterator = m_actionToSortKeyMap.find(actionIdentifier);
+        if (actionIterator == m_actionToSortKeyMap.end())
+        {
+            return 0;
+        }
+
+        return actionIterator->second;
+    }
```

Comment:
question: Is returning `0` here a good idea? Could `0` be a legitimate value? We could instead potentially return an `optional` to handle the case the value isn't found?

Reply:
Okay awesome thanks, appreciate the info 👍